### ISSUE TITLE
Spells are cast from behind player

### ISSUE THREAD

**Issue by MetricExpansion** [mwilsnd/SkyrimSE-SmoothCam#21]
First of all, great work on the mod. I'm loving it so far! I've been wanting something like this for years and I was overjoyed to stumble upon this mod.

While it's been great so far, I have run into an issue with spellcasting. It seems that spells are being cast from slightly behind my player. This becomes more extreme when my camera is zoom level is more zoomed out, and it gets particularly bad while my zoom level is zoomed out but where the camera is forced to a closed position to due to a wall. This also prevents me from casting spells high into the air, as the spells will just hit the ground beneath me. These issues go away in first-person mode.

For what it's worth, I am using the 1.5.73 runtime, which I know isn't officially supported, but it might be worth trying to see if you can reproduce the issue. I don't have any other mods that affect my camera.

I've made a small video of the issue here: https://www.youtube.com/watch?v=OHi8wVT0yj0

**Comment by mwilsnd** [comment]
Interesting, thus far I haven't been able to replicate this. At first I thought it might be the downward aiming patch in EngineFixesSSE somehow conflicting with my own patch to projectile flight paths in such a way as to make everything appear to work, but turning that fix off didn't change anything.

A list of mods you're using would be helpful in tracking down what is actually causing this. Thanks for reporting this issue.

**Comment by MetricExpansion** [comment]
Okay, so I did a little experimenting. You mentioned the [SSE Engine Fixes](https://www.nexusmods.com/skyrimspecialedition/mods/17230) mod. I tested disabling that mod completely and, in fact, that *did* fix my issue. After enabling the Engine Fixes again, I tried to disable the ArcheryDownwardAiming fix in EngineFixes.toml, and now my spellcasting works correctly!

So while you seem to be running fine with both mods, it seems to be the case for me that enabling that specific fix from SSE Engine Fixes causes the issue for me. Perhaps it's just something funny going on with the fact that I'm using an older runtime (1.5.73)? 

Here's my mod list: [modlist.txt](https://github.com/mwilsnd/SkyrimSE-SmoothCam/files/5361382/modlist.txt)
